server/controller/UserController.js

```diff
@@ -660,6 +661,28 @@ router.get(
     }
   }
 );
+// @desc Retrieves all user e-mails
+// @route GET /users/emails/:includeAll
+// Only moderator
+
+router.get(
+  "/emails/:includeAll",
+  authToken,
+  checkParamIsBoolean("includeAll"),
+  async (req, res, next) => {
+    try {
+      if (await requesterIsNotModerator(req, res)) return;
```

Comment:
I'm not sure if the moderator should have this power. Especially with the multiple mails problem that should be verified manually before sending a mail…

Reply:
Me neither^^

But the problem is on client-side not on server-side. Therefore i wouldn't change the server implementation, but would remove it from the moderator options in the AdminDashboard

Reply:
Or change them both. A powerful moderator could find this out 🤪

Reply:
A powerful mod wouldn't have a problem with verifying emails neither